### PR TITLE
Home

### PR DESCRIPTION
**Pull request by mirisuzanne**
Arthur

- [Trello card](https://trello.com/c/NhQf3F90/27-i-want-a-home-page)
- [Deploy Preview](https://deploy-preview-15--oddleventy.netlify.com/)

## Description
The home page!

I think some of the content/organization is still up for debate, but if we have roughly the right pieces to work with, I'd like to merge and iterate.

For now the logo blocks are basically placeholder - I'd like to get all the projects merged in and cleaned up, so we can pull from live project data to extend those with links and more information. But this branch provides a lot of infrastructure that will be useful in migrating more pages over.

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[I want a home page](https://trello.com/c/NhQf3F90/27-i-want-a-home-page)

File: content/_includes/page/intro.njk [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% import 'utility.macros.njk' as utility %}
+
+{% if summary or action %}
+  <div id='intro'>
```
Comment (stacyk) [inline content/_includes/page/intro.njk]: should we use double quotes in html? I see it as double in other files when using straight up html.

File: content/work/coachhub.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
+---
+title: 'Case Study: CoachHub'
+image:
+  src: projects/coachhub.jpg
+  position: top
+client:
+  name: &client ORCAS
```
Comment (stacyk) [inline content/work/coachhub.md]: these &word pairings are curious ... but I'm going to guess this is done to be able to use some of the data throughout what you've built already (knowing we haven't built out page or case study templates yet?) or is this the way we need to enter some info now?

File: src/scss/config/_scale.scss [hunk 3]
```diff
@@ -50,6 +50,7 @@ $spacing-sizes: (
   'newline': '#gutter',
   'gutter-plus': calc(var(--gutter) * 1.5),
```
Comment (stacyk) [inline src/scss/config/_scale.scss]: I love the -plus idea since there isn't a good name for 1.5 of a thing.

File: src/scss/config/_scale.scss [hunk 4]
```diff
@@ -71,7 +72,7 @@ $spacing-sizes: (
 /// @sizes {ruler}
 /// @group config-scale
 $theme-sizes: (
-  'body-inline': calc(var(--quarter-shim) + 4vw),
+  'page-margin': calc(var(--quarter-shim) + 4vw),
```
Comment (stacyk) [inline src/scss/config/_scale.scss]: I like this label better

File: src/scss/layout/_header.scss [hunk 5]
```diff
@@ -1,29 +1,48 @@
...
+    margin-bottom: 0;
+    padding-left: var(--page-margin);
+  }
+
+  #meta,
+  #intro {
+    padding: 0 var(--page-margin);
+  }
+
   @media (min-width: 35em) and (min-height: 35em) {
```
Comment (stacyk) [inline src/scss/layout/_header.scss]: should we use the new media-page variable here? I know we are using rems when it is declared though so maybe there is a difference?

File: src/scss/patterns/_sections.scss [hunk 6]
```diff
@@ -1,93 +1,70 @@
...
-[divider-title] {
-  font-size: inherit;
+@use '../../../node_modules/accoutrement/sass/tools';
+@use '../config/scale';
+@use 'sass:map';
+
+// Main Actions
+// ------------
+/// Main actions are large block links,
+/// used atthe end of a section
```
Comment (stacyk) [inline src/scss/patterns/_sections.scss]: `at the` needs a space

File: src/scss/patterns/_sections.scss [hunk 7]
```diff
@@ -1,93 +1,70 @@
...
+/// to represent themain call to action of that section.
```
Comment (stacyk) [inline src/scss/patterns/_sections.scss]: `the main` needs a space

File: content/_includes/contact.macros.njk [hunk 8]
```diff
@@ -48,3 +48,110 @@
...
+                {{ 'checked' if (group.key in interests) else '' }}
+                >
+              {{ group.label }}
+            </label>
+          </li>
+        {% endfor %}
+      </ul>
+
+      <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+      <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_80219aa68d7bad77b9fd2eb93_7c27f7fb9a" tabindex="-1" value=""></div>
```
[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/_includes/contact.macros.njk]: seems like marking it as hidden should cover that?

File: content/_includes/embed.macros.njk [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+{%- set style = {
+  '--width': width,
+  '--height': height
+} -%}
+<div data-place {{ utility.style_if(style) }}>{{ content | safe }}</div>
+{%- endmacro %}
+
+
+{#
+  Private
```
[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/_includes/embed.macros.njk]: It's a private macro (not available outside the file). Probably doesn't need the label, but that will get cleaned up in the "docs" branch I'm working on.

File: content/work/coachhub.md (hunk 2 above)
Comment (mirisuzanne) [reply content/work/coachhub.md]: this is a [feature of YAML](https://github.com/cyklo/Bukkit-OtherBlocks/wiki/Aliases-(advanced-YAML-usage)) that we haven't used much before, but it allows us to define something once, give it a name, and then re-use it somewhere else in the YAML. I started using it to add client information at the root level, while also attaching it to individual people (e.g. `venue: *client`) –  and to reference people (e.g. `&sara`) from multiple quotes (`<<: *sara`).

File: src/scss/layout/_header.scss (hunk 5 above)
Comment (mirisuzanne) [reply src/scss/layout/_header.scss]: Mainly I was thinking that they aren't really tied together. We could change the size we want for a media-page, and there's no reason for these media-queries to change along with that. I'm also trying to be a bit more careful about what belongs as a global pattern, and what really is just a component one-off. These numbers aren't really tied to anything else - just what makes sense for the header layout.

File: src/scss/patterns/_sections.scss (hunk 6 above)
Comment (mirisuzanne) [reply src/scss/patterns/_sections.scss]: macbook keyboards areawesomeee..

**Pull request merged**